lib/backburner/index.js

```diff
@@ -28,7 +28,7 @@ export default function Backburner(queueNames, options) {
 
   var _this = this;
   this._boundClearItems = function() {
-    clearItems();
+    clearItems.apply(_this, arguments);
```

Comment:
> The whole reason for this regression was changing to avoid bind, right?

TBH, I am unsure, I have not dug into the original reasoning for the change...